tests/Workleap.DotNet.CodingStandards.Tests/PackageFixture.cs

```diff
@@ -0,0 +1,56 @@
+using System.Diagnostics;
+using CliWrap;
+using Meziantou.Framework;
+using Workleap.DotNet.CodingStandards.Tests.Helpers;
+
+namespace Workleap.DotNet.CodingStandards.Tests;
+
+public sealed class PackageFixture : IAsyncLifetime
+{
+    private readonly TemporaryDirectory _packageDirectory = TemporaryDirectory.Create();
+
+    public FullPath PackageDirectory => _packageDirectory.FullPath;
+
+    public async Task InitializeAsync()
+    {
+        var nuspecPath = PathHelpers.GetRootDirectory() / "Workleap.DotNet.CodingStandards.nuspec";
+        string[] args = ["pack", nuspecPath, "-ForceEnglishOutput", "-Version", "999.9.9", "-OutputDirectory", _packageDirectory.FullPath];
```

Comment:
Could we use our own coding standard here? wink wink

Reply:
What do you mean? The one built on the CI?

Reply:
Yeah I meant consuming our own package in this C# project. You already did it via Directory.Build.props!